### PR TITLE
Minor text changes to /open, /join and /mute slash commands

### PR DESCRIPTION
**Pull request by jasonblais**
#### Summary
Minor text changes to slash commands. Discussed in UX channel.

1) In autocomplete entry for /mute, /join and /open slash commands, add `~` in front of [channel] similar to `@[username]` to make the channel linking more discoverable

image

2) Update error message when channel cannot be found in /mute command.

#### Ticket Link
N/A

#### Checklist
- [X] Includes text changes and localization file ([.../i18n/en.json](https://github.com/mattermost/mattermost-server/blob/master/i18n/en.json)) updates

### CODE REVIEW

**Comment by jasonblais** [comment]
WIP label - want to test my changes are correct.

**Comment by jasonblais** [comment]
HTML isn't rendering properly. I know how to fix it for jsx files, but not sure how to fix it on the go side..

image

File: i18n/en.json [hunk 1]
```diff
@@ -1020,15 +1020,15 @@
...
+    "translation": "~[channel]"
   },
   {
     "id": "api.command_mute.name",
     "translation": "mute"
   },
   {
     "id": "api.command_mute.error",
-    "translation": "Could not find the channel {{.Channel}}."
+    "translation": "Could not find the channel {{.Channel}}. Please use the <a href=\"https://about.mattermost.com/default-channel-handle-documentation/\">channel handle</a> to identify channels."
```
Comment (lieut-data) [inline i18n/en.json]: If I traced the code correctly, I think this translated bit of text is used as the `Message` of a `Post`, so I'm pretty sure you can just write this as Markdown.

**Comment by jasonblais** [comment]
Awesome, thank you @lieut-data - that did the trick.

If you can help review & merge, that would be good..many thanks

**Pull request merged**